docs/design_docs/generalized_horizontal_interpolation.md

```diff
@@ -5,6 +5,9 @@ Generalized Horizontal Interpolation in MPAS-Analysis
 Xylar Asay-Davis <br>
 date: 2017/01/19 <br>
 </h2>
+
+Horizontal interpolation and related utilities for remapping are now in the [pyremap repo](https://github.com/MPAS-Dev/pyremap/).
```

Comment:
This design doc is about the python-based horizontal interpolation, rather than remapping more generally.  The python-based remapping still requires that mapping files be provided, and that is outside the scope of the design doc.  That's why I think this statement is a little out of place.  But if it points people in the right direction, it does no harm.